lab_prog_2.cpp

```diff
@@ -5,11 +5,11 @@ int main()
     int n,isum=0,i;
     cout<<"enter the value of n ";
     cin>>n;
-    for(i=1;i<=n;i++)
```

Comment:
If we gave input as 1 what will be the output?
as per your logic it will print 2.
Whereas output should be 1

Reply:
if we give input as 1 then it will print 1 as output. only 1 time for loop will be executed.